docker-compose.yml

```diff
@@ -98,9 +99,37 @@ services:
     container_name: kib01
     ports:
       - 5601:5601
+    depends_on:
+      - "es01"
+      - "es02"
+      - "es03"
+      - "ent-search"
     environment:
       ELASTICSEARCH_URL: http://es01:9200
       ELASTICSEARCH_HOSTS: '["http://es01:9200","http://es02:9200","http://es03:9200"]'
+      ENTERPRISESEARCH_HOST: http://ent-search:3002
+    networks:
+      - elastic
+
+  ent-search:
+    image: docker.elastic.co/enterprise-search/enterprise-search:7.11.1
+    restart: unless-stopped
+    depends_on:
+      - "es01"
+      - "es02"
+      - "es03"
+    environment:
+      - "JAVA_OPTS=-Xms512m -Xmx512m"
+      # - "ENT_SEARCH_DEFAULT_PASSWORD=changeme"
+      # - "elasticsearch.username=elastic"
+      # - "elasticsearch.password=changeme"
+      - "elasticsearch.host=http://es01:9200"
+      - "allow_es_settings_modification=true"
+      - "secret_management.encryption_keys=[4a2cd3f81d39bf28738c10db0ca782095ff\
+        ac07279561809eecc722e0c20eb09]"
+      - "elasticsearch.startup_retry.interval=15"
+    ports:
+      - 127.0.0.1:3002:3002
```

Comment:
I thought this could help me, but didn't use it after all. I don't actually know how powerful it can be, but Elastic documentation referred to it a lot :) Can be taken away if not needed. 

![image](https://user-images.githubusercontent.com/389204/146202582-67678950-41ea-4b1c-8908-9fdd26ef92ed.png)

Reply:
In Kibana: 
![image](https://user-images.githubusercontent.com/389204/146204301-47682352-e59c-49f9-9c6e-ef2083678cce.png)

Reply:
Personally I think we shouldn't add stuff unless we are pretty sure it is needed. But I don't know this product at all, so you are better to decide whether it should be added or not 🙂 If you decide to leave it, I think it would be nice if it was somehow mentioned in the README.